### ISSUE TITLE
Can AWSFileStore be used in a client application (using para-client)?

### ISSUE THREAD

**Issue by alexvillalta** [Erudika/para#42]
I am trying to add file uploads to a client application (Scoold, which uses para-client.) Is there a method in the ParaClient API that can use AWSFileStore to upload files to an Amazon S3 bucket, or must the client application integrate para-server to have access to AWSFileStore (i.e. by directly importing com.erudika.para.storage.AWSFileStore?)

**Comment by albogdano** [comment]
No, the file upload part of Para is not exposed through the REST API, so the clients don't support it. It's a low priority feature for now. It shouldn't be too hard to just work around that by calling S3 directly from Scoold controllers using the S3 SDK for Java.
